docs/SchemaApi.md

```diff
@@ -342,7 +379,7 @@ with sajari_client.ApiClient(configuration) as api_client:
     # and optional values
     try:
         # List schema fields
-        api_response = api_instance.list_schema_fields(collection_id, page_size=page_size, page_token=page_token)
+        api_response = api_instance.list_schema_fields(collection_id, parent=parent, account_id=account_id, page_size=page_size, page_token=page_token)
```

Comment:
This isn't a breaking change?

Reply:
Nope because they are optional named arguments. Callers don't have to pass them. And in Python they are in any order too.